.yarnrc.yml

```diff
@@ -0,0 +1,9 @@
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
+    spec: '@yarnpkg/plugin-workspace-tools'
+  - path: .yarn/plugins/@yarnpkg/plugin-compat.cjs
+    spec: '@yarnpkg/plugin-compat'
```

Comment:
This shouldn't be needed if we're using node-modules above. The PNP linker is disabled, so this package shouldn't have any effect.